app/controllers/CandidateDetailsController.scala

```diff
@@ -16,7 +16,7 @@
 
 package controllers
 
-import model.Exceptions.{CannotUpdateContactDetails, CannotUpdateRecord, ContactDetailsNotFound, PersonalDetailsNotFound}
+import model.Exceptions._
 import model.command.UpdateGeneralDetails
```

Comment:
I think UpdateGeneralDetails is a bit misleading, it is also used in the find method, I think a better name would be command.GeneralDetails or maybe command.PersonalDetails